api-reference/10 UI Components/dxAccordion/1 Configuration/deferRendering.md

```diff
@@ -8,3 +8,14 @@ default: true
 Specifies whether to render the panel's content when it is displayed. If **false**, the content is rendered immediately.
 
 ---
+
+---
+##### Angular
+
+If you set the `deferRendering` property to **true**, wrap the required content in the Angular named [template](/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Templates).
```

Comment:
```suggestion
If you set the `deferRendering` property to **true**, wrap the content in the Angular [template](/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Templates).
```

Reply:
In this description I need to focus attention on the **named templates**. So the description should be as follows: ... 
![named_templates](https://user-images.githubusercontent.com/112853396/217197486-e757d501-ad2d-4794-b52a-02ab7657e8ba.png)
wrap the content in the Angular **named template**.

Reply:
Ok, that is fine then